app/components/checkbox.gjs

```diff
@@ -0,0 +1,22 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import checked from 'frontend-timekeeper/modifiers/checked';
+import { on } from '@ember/modifier';
+
+export default class CheckboxComponent extends Component {
+  
+  handleClick = (event) => {
+    this.args.onChange?.(event.target.checked, event);
+  }
+
+  <template>
+    {{! template-lint-disable require-input-label }}
+    <input
+      type='checkbox'
+      class='w-4 h-4 text-red-700 bg-gray-100 border-gray-300 rounded focus:ring-red-500 focus:ring-2'
+      {{checked @isChecked}}
```

Comment:
Can't  you just pass `checked` as attribute. Is the modifier required?

Reply:
I thought because checked was a boolean attribute (`true` when present) this would not work, but turns out it does. Fixed in [3422fe8](https://github.com/redpencilio/frontend-timekeeper/pull/4/commits/3422fe8380c88ea7e42f97e3e84622c73cc65d5d)